### DISCUSSION TITLE
Promoting the addon

### DISCUSSION THREAD

**Discussion by Vilhelm-Ian** [mortii/anki-morphs#101] [General]
I think the addon is more than 1:1 with morphman. And improved in many ways(speed, usability). And other than having the same core features it has extra features.

I think we should write a post and share it on reddit(/r/anki, /r/refold) 

And @HQYang1979 can start making the video he wanted to

**Comment by HQYang1979** [comment]
I've been writing the script for the video.

And I am most looking forward to the implementation of spaCy, that works on Korean, German, English, Spanish etc...

I thought spaCy would be implemented in 0.80...

**Reply by mortii** [reply]
> I thought spaCy would be implemented in 0.80...

Other useful (but unforeseen) features skipped the queue.

**Comment by mortii** [comment]
@Vilhelm-Ian I'm glad you think it is worth promoting :) 

That being said, I kinda want to wait until after the alpha is over.  Backwards compatibility breaking is a terrible user experience, and the spacy implementation will be the biggest breaker of them all.

If everything goes smoothly then the alpha might be over in ~1 month hopefully. It could  be way longer of course, but that's my intuition right now.

**Comment by mortii** [comment]
@Vilhelm-Ian @HQYang1979 

AnkiMorphs is now out of alpha, so any promotions/videos are fine to do now :)

I suspect the setup of AnkiMorphs is too complicated for most people, so I don't expect the addon to be very popular. But that's okay, It's been satisfying to make, and I have have gotten a lot of use out of it, so I consider it a success regardless of how many people use it :v: 

Thank you guys for even considering doing something like this, it's an honour!

**Comment by Vilhelm-Ian** [comment]
Anki-Morphs

Anki-Morphs is the spiritual successor of the addon MorphMan.
 - The goal of the addon is to sort your new cards so you will be shown a sentence where you will have only one unknown word in the sentence(i+1 sentence) following the [Input Hypothesis](https://en.wikipedia.org/wiki/Input_hypothesis#:~:text=The%20input%20hypothesis.,next%20stage%20of%20language%20acquisition). 
 - The benefits are 
     - you don't have to manually sentence mind cards.
     -  You can also use frequency lists so the order of cards you will be shown will be most beneficial for your language learning journey
     
Improvements over Morphman
- Compatbile with modern versions of anki
- It also makes the UI much much more friendly
-  It also uses spacy so languages without spaces can be broken down to their words.

To install the addon visit the following [link](https://ankiweb.net/shared/info/472573498)
To learn how to use it read this [guide](https://mortii.github.io/anki-morphs/user_guide/usage.html)
To contribute visit this [repo](https://github.com/mortii/anki-morphs)

**Reply by aleksejrs** [reply]
> * you don't have to manually sentence mind cards.

?!

> It also makes the UI much much more friendly

Last I checked here, it worked with only one field, which is an important difference (bad to me).

**Reply by mortii** [reply]
@Vilhelm-Ian Nice :+1: 

> Last I checked here, it worked with only one field, which is an important difference (bad to me).

@aleksejrs There are many things that have been simplified compared to morphman (for better or worse), the readability analyzer in particular is not similar at all. I'm sure many will prefer the morphman implementation, and that is completely fine--after all, It's not possible to please everyone. That being said, I much prefer the design and UI of ankimorphs, so this point just boils down to preference imo.
